packages/rspack/src/config/types.ts

```diff
@@ -2058,7 +2058,10 @@ export type OptimizationRuntimeChunk =
 			name?: string | ((value: { name: string }) => string);
 	  };
 
-export type OptimizationSplitChunksNameFunction = (module?: Module) => unknown;
+export type OptimizationSplitChunksNameFunction = (
+	module?: Module,
+	chunks?: Chunk[]
+) => unknown;
```

Comment:
I don't know why the `module` parameter was optional in the previous definition. We can also add the missing `cacheGroupKey` parameter.
```suggestion
export type OptimizationSplitChunksNameFunction = (
	module: Module,
	chunks: Chunk[],
	cacheGroupKey: string
) => string | undefined;
```

Reply:
Done as suggested :slightly_smiling_face: 
As @chenjiahan pointed out, I also updated the zod schema, which I hadn't known about.
I ran `npm run api-extractor:local`, but that produced _a lot_ of diffs, idk if that seems right.